utils/github/getDecodedData.ts

```diff
@@ -11,11 +11,21 @@ const getDecodedData = async (repoFullName, headBranch, path, accessToken) => {
     path,
     accessToken
   )
-
-  if ((response?.response?.status || 0) == 404) {
-    throw new ContentNotFoundError(
-      'Content not found. Your fork may have been deleted.'
-    )
+  
+  const errorStatus = response?.response?.status || 200
```

Comment:
I ran into an issue last week where we were receiving a successful 201 code, but we were treating it as an error. We may want to abstract this into a helper: https://github.com/tinacms/tinacms.org/blob/master/open-authoring/github/api/index.js#L103